merino/featureflags.py

```diff
@@ -45,17 +38,20 @@ class BucketingScheme(str, Enum):
 class FeatureFlag(BaseModel):
     """Model representing a feature flag."""
 
-    enabled: Enabled
+    enabled: confloat(ge=0.0, le=1.0)  # type: ignore [valid-type]
```

Comment:
I would recommend using the 3.9+ `Annotated` type here. It is what `confloat` uses now and is recommended for describing type constraints. You should also be able to drop the mypy type ignore comment.

```suggestion
    enabled: Annotated[int, Field(gt=0.0, lte=1.0)]
```

https://docs.pydantic.dev/latest/usage/types/custom/

Reply:
Thanks for the suggestion @quiiver !

Reply:
oh yes that's right